notebooks/MAST/find_eclipse_candidates/find_eclipse_candidates.ipynb

```diff
@@ -0,0 +1,754 @@
+{
```

Comment:
<p>These are nice exercises!</p><p> </p><p>In item 1, might also suggest sorting by primary mass (M1) or the velocity semi-amplitude (K).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>